packages/components/input/src/use-input.ts

```diff
@@ -330,7 +332,7 @@ export function useInput<T extends HTMLInputElement | HTMLTextAreaElement = HTML
         "data-focus-visible": dataAttr(isFocusVisible),
         "data-focus": dataAttr(isFocused),
         className: slots.inputWrapper({
-          class: clsx(classNames?.inputWrapper, !!inputValue ? "is-filled" : ""),
+          class: clsx(classNames?.inputWrapper, !isEmpty(inputValue) ? "is-filled" : ""),
```

Comment:
```suggestion
         class: clsx(classNames?.input, isFilled ? "is-filled" : ""),
```

Reply:
"classNames?.inputWrapper" should not be "classNames?.input".